packages/client/tests/functional/create-many/tests.ts

```diff
@@ -0,0 +1,102 @@
+import * as faker from 'faker'
```

Comment:
IIRC, fresher maintained version of the package is `@faker-js/faker`

Reply:
(Idea: I found https://github.com/snaplet/copycat to be more maintainable/readable, made by our friends at snaplet).

Reply:
Oh, wow, I actually worked with the guy who made it, and I know that he knows a lot about generating very complex test data deterministically. So yea, changing my suggestion to `copycat` as well.

Reply:
Ive gone with `@faker-js/faker` simply because copycat currently has 9 weekly downloads and are we sure we want to trust a lib with this little usage? It looks as thoe copycat just uses and exports faker anyway!